.github/workflows/sync-ceremony-to-main.yml

```diff
@@ -63,3 +62,82 @@ jobs:
           pr_title: "Merge ceremony branch ${{ inputs.branch || github.ref_name }} into ${{ github.event.repository.default_branch }}"
           pr_body: "Merge ceremony branch to main"
           pr_reviewer: asraa,dlorenc,haydentherapper,joshuagl,kommendorkapten
```

Comment:
should this reviewer list be a team that we can define via pulumi vs having this hard-coded?

Reply:
We could also omit it entirely, as automation should handle reviewing.

Reply:
I agree, though since that was there before, I'd normally suggest a different PR for it. But I can also just remove it in a separate commit if that seems better.

Reply:
Id say to leave it for a later change.